__tests__/business-text/business-text.mutation.test.js

```diff
@@ -0,0 +1,292 @@
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+
+const { BUSINESS_TEXT_NOT_FOUND } = require('../../error-messages/business-text.messages');
+const { newBusinessText, updatedBusinessText, notExistBusinessTextId } = require('./business-text.variables');
+require('dotenv')
+  .config();
+
+let businessText = null;
+let businessTextId = '';
+
+describe('Business text mutations test', () => {
+
+  test('#1 should add business text to database', async () => {
```

Comment:
I think you also need the test where you try to add business page with already existing code and that proper error message is returned